test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -124,6 +125,70 @@ def _run_test(m):
             _run_test(script_model)
         _run_test(model)
 
+    def run_large_model_test(self, model, input, rtol=0.001, atol=1e-7,
```

Comment:
run_model_test_with_external_data?

Reply:
Actually, can we merge run_model_test and run_large_model_test together by adding one flag. Since major logic should be the same, except the temporary file/directory part.

Reply:
1) Agreed. Done. 
2) I actually considered that but there are some differences between the two (which sometimes can be subtle) method bodies to a point where I feel it will be better to keep them apart for clarity. If it is OK with you, I would like to keep them separate for now.